### DISCUSSION TITLE
Who's still on .NET Framework?

### DISCUSSION THREAD

**Discussion by louthy** [louthy/language-ext#1280] [Polls]
It's now getting to the point where sticking to .NET Standard 2.0 is holding back this library.  I realise this poll isn't likely to get seen by many people, so it might not be representative, but could you let me know whether you need .NET Framework support from this library.

**Comment by BrunoJuchli** [comment]
We're planning to replace all of our .net Framework with .net "core", but unfortunately it'll take about another year until we're finished with that :-/

**Comment by MikeCook9994** [comment]
My team has a collection of microservices that are still on .NET Framework 472. I actively encourage everyone to drop support for it because it's ammunition that I can use to convince leadership to let us migrate finally.

**Comment by louthy** [comment]
Thanks for the votes and feedback.  The decision has been made to move to .NET8.0 for the upcoming `5.0.0` release -- primarily to leverage:

* The static-interface-members features for [Higher Kinds](https://paullouth.com/higher-kinds-in-c-with-language-ext/)
* Collection expressions (`Seq<int> xs = [1,2,3]`)
* Lambda improvements
* Performance improvements

The enabling of [trait](https://en.wikipedia.org/wiki/Trait_%28computer_programming%29) functionality with the static-interface members is the big one and has led to a massive refactor -- but it makes the library 1000x more powerful and 1000x easier to maintain.  That's something I can't ignore.
